### PR TITLE
OCM-5645 | fix: revise help message for `rosa grant/revoke` command

### PR DESCRIPTION
**Pull request by marcolan018**
https://issues.redhat.com/browse/OCM-5645
Signed-off-by: marcolan018 <llan@redhat.com>

### CODE REVIEW

**Review by willkutler — APPROVED** [review]

File: cmd/grant/user/cmd.go [hunk 1]
```diff
@@ -87,9 +87,9 @@ func run(_ *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 	if username == idp.ClusterAdminUsername {
-		r.Reporter.Errorf("Username '%s' is not allowed. It's must in '%s' group. "+
+		r.Reporter.Errorf("Username '%s' is preserved for `rosa create/delete admin` command. "+
```
Comment (gdbranco) [inline cmd/grant/user/cmd.go:90]: Should the term be reserved instead of preserved?

Comment (marcolan018) [reply cmd/grant/user/cmd.go]: @gdbranco 
thanks for the commments
can you take another look on the fixed one? Thanks

**Pull request merged**